android/gradle.properties

```diff
@@ -1 +1,3 @@
 org.gradle.jvmargs=-Xmx1536M
+android.useAndroidX=true
```

Comment:
Really nitpicky request, but can you ensure this file is sorted alphabetically?

Reply:
Sure - that is a minor fix.